.makefile

```diff
@@ -0,0 +1,62 @@
+# This file is used for common development targets that can be done with
+# needing the cumbersome bootstrapping process.
+#
+# You can use it like this:
+#
+#   make -f .makefile indent
+#
+# If you copy or link this file to `GNUmakefile` then you can just do:
+#
+#   make indent
+#
+# When copied to `GNUmakefile`, this file is can also be used for bootstrapping
+# Makefile targets. Since GNUmakefile is loaded before Makefile, we do the
+# bootstrapping tasks need to get a Makefile first, then we use the Makefile to
+# make our target.
+
+# Remind user when they are using GNUmakefile:
+ifeq ($(lastword $(MAKEFILE_LIST)),GNUmakefile)
+    $(info *** NOTE: GNUmakefile in use. ***)
+endif
+
+MAKE_TARGETS := \
+	all \
+	all-am \
+	all-recursive \
+	install \
+	test \
+	test-all \
+	test-suite \
```

Comment:
You mean to proxy? I added my own .makefile distclean rule (line 60) because the Makefile distclean doesn't work.

So, no, I wouldn't proxy that rule until we fix it.